src/libcprover-rust/src/lib.rs

```diff
@@ -0,0 +1,129 @@
+use cxx::{CxxVector, CxxString};
+#[cxx::bridge]
+pub mod ffi {
+
+    unsafe extern "C++" {
+        include!("libcprover-cpp/api.h");
+        include!("include/c_api.h");
+
+        type api_sessiont;
+
+        // API Functions
+        fn new_api_session() -> UniquePtr<api_sessiont>;
+        fn get_api_version(&self) -> UniquePtr<CxxString>;
+        fn load_model_from_files(&self, files: &CxxVector<CxxString>);
+        fn verify_model(&self);
+        fn validate_goto_model(&self);
+        fn drop_unused_functions(&self);
+
+        // Helper/Utility functions
+        fn translate_vector_of_string(elements: Vec<String>) -> &'static CxxVector<CxxString>;
+        fn get_messages() -> &'static CxxVector<CxxString>;
+    }
+
+    extern "Rust" {
+        fn print_response(vec: &CxxVector<CxxString>);
+    }
+}
+
+fn print_response(vec: &CxxVector<CxxString>) {
+    let vec: Vec<String> = vec
+        .iter()
+        .map(|s| s.to_string_lossy().into_owned())
```

Comment:
Why is `into_owned` needed? Is this an expectation of the API that ownership move here?